pennylane/qcut/__init__.py

```diff
@@ -13,6 +13,51 @@
 # limitations under the License.
 """
 This subpackage contains quantum function transforms for cutting quantum circuits.
+
+.. currentmodule:: pennylane
+
+Transform for circuit cutting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :func:`~.cut_circuit` transform accepts a QNode and returns a new function that cuts the original circuit,
+allowing larger circuits to be split into smaller circuits that are compatible with devices that
+have a restricted number of qubits.
+
+.. autosummary::
+    :toctree: api
+
+    ~cut_circuit
+
+The :func:`~.cut_circuit_mc` transform is designed to be used for cutting circuits which contain :func:`~.sample`
+measurements and is implemented using a Monte Carlo method. Similarly to the :func:`~.cut_circuit`
+transform, this transform accepts a QNode and returns a new function that cuts the original circuit.
+This transform can also accept an optional classical processing function to calculate an
+expectation value.
+
+.. autosummary::
+    :toctree: api
+
+    ~cut_circuit_mc
+
+There are also low-level functions that can be used to build up the circuit cutting functionalities:
+
+.. autosummary::
+    :toctree: api
+
+    ~qcut.tape_to_graph
+    ~qcut.replace_wire_cut_nodes
+    ~qcut.fragment_graph
+    ~qcut.graph_to_tape
+    ~qcut.expand_fragment_tape
+    ~qcut.expand_fragment_tapes_mc
+    ~qcut.qcut_processing_fn
+    ~qcut.qcut_processing_fn_sample
+    ~qcut.qcut_processing_fn_mc
+    ~qcut.CutStrategy
+    ~qcut.kahypar_cut
+    ~qcut.place_wire_cuts
+    ~qcut.find_and_place_cuts
+
```

Comment:
If not, we could just go with something super short like:

```suggestion
Cutting Circuits
----------------

Circuit cutting allows you to replace a circuit with ``N`` wires by a set
of circuits with less than ``N`` wires (see also
`Peng et. al <https://arxiv.org/abs/1904.00102>`_). Of course this comes
with a cost: the smaller circuits require a greater number of device
executions to be evaluated.

In PennyLane, circuit cutting for circuits that terminate in expectation values
can be activated by positioning :class:`~.pennylane.WireCut` operators at the
desired cut locations, and by decorating the QNode with
the :func:`~.pennylane.cut_circuit` transform.

Cut circuits remain fully differentiable, and the resulting circuits can be
executed on parallel devices if available. Please see the
:func:`~.pennylane.cut_circuit` documentation for more details.

.. note::

    Simulated quantum circuits that produce samples can be cut using
    the :func:`~.pennylane.cut_circuit_mc`
    transform, which is based on the Monte Carlo method.
    
Automatic cutting
-----------------

PennyLane also has experimental support for automatic cutting of circuits ---
that is, the ability to determine optimum cut location without explicitly
placing :class:`~.pennylane.WireCut` operators. This can be enabled by using the
``auto_cutter`` keyword argument of :func:`~.pennylane.cut_circuit`; refer to the
function documentation for more details.
```

Reply:
I don't think we have anything else than what is already on the page, I am good with your suggestion.

Reply:
I keep this thread open in case @trbromley you have something to add.

Reply:
Nice, I left a few comments but overall looks great to me!